wagtail_streamfield_migration_toolkit/test/tests/test_streamchangedetector.py

```diff
@@ -165,3 +176,27 @@ def test_nested_struct_child_renamed(self, mock_class):
         ):
             self.assertEqual(expected_block_path, block_path)
             self.assertRenameOperationEqual(expected_operation, operation)
```

Comment:
Is it worth checking if the mock questioner method was called for each of these cases?